Doctrine migration is not up to date with the entity mappings

After migrating migration `BitBag\SyliusWishlistPlugin\Migrations\Version20201029161558` 
and checking schema by following commad
```bash
bin/console doctrine:schema:validate
```
It will return an error
```text
[ERROR] The database schema is not in sync with the current mapping file.
```
and after checking for missing SQL queries
```bash
bin/console doctrine:schema:update --dump-sql
```
returns:
```
 The following SQL statements will be executed:

     ALTER TABLE bitbag_wishlist_product DROP FOREIGN KEY FK_3DBE67A0FB8E54CD;
     ALTER TABLE bitbag_wishlist_product ADD CONSTRAINT FK_3DBE67A0FB8E54CD FOREIGN KEY (wishlist_id) REFERENCES bitbag_wishlist (id) ON DELETE CASCADE;
```
The next migration file should be created?

--- a/src/Migrations/Version20210428130552.php
+++ b/src/Migrations/Version20210428130552.php
@@ -1,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * @deprecated Since bitbag/wishlist-plugin 2.0: Doctrine migrations existing in a bundle will be removed, move migrations to the project directory.
+ */
+final class Version20210428130552 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add missing constraint to bitbag_wishlist_product';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE bitbag_wishlist_product DROP FOREIGN KEY FK_3DBE67A0FB8E54CD');
+        $this->addSql('ALTER TABLE bitbag_wishlist_product ADD CONSTRAINT FK_3DBE67A0FB8E54CD FOREIGN KEY (wishlist_id) REFERENCES bitbag_wishlist (id) ON DELETE CASCADE');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE bitbag_wishlist_product DROP FOREIGN KEY FK_3DBE67A0FB8E54CD');
+        $this->addSql('ALTER TABLE bitbag_wishlist_product ADD CONSTRAINT FK_3DBE67A0FB8E54CD FOREIGN KEY (wishlist_id) REFERENCES bitbag_wishlist (id)');
+    }
+}

--- a/tests/Application/.env
+++ b/tests/Application/.env
@@ -12,7 +12,7 @@ APP_SECRET=EDITME
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
-DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.5
+DATABASE_URL=mysql://root@127.0.0.1/sylius_wish_list_plugin_%kernel.environment%?serverVersion=5.7
 ###< doctrine/doctrine-bundle ###
 
 ###> symfony/swiftmailer-bundle ###

